### ISSUE TITLE
Update the Wiki with the current GitHub footprint

### ISSUE THREAD

**Issue by ActionAnalytics** [bcgov/cloud-pathfinder#2130]
**Describe the Issue**
The wiki in Teams has some placeholders for Public and Private cloud lists of github repos, but they are a year out of date.  The community of BC Gov cloud users will be better served by lowering the risk of continuity problems if key people on the team leave by having periodic update of the GitHub repo footprint.

**Additional Context**
Add any other context, attachments or screenshots

**Acceptance Criteria**
- ask PO questions in Teams any time
- go into the Wiki in Teams under "CPF Internal Team" and update the wiki with
  - Current repos
  - A one-line sentence beside the repo entry in the wiki summarizing the purpose (highlight ones that aren't clear in yellow and we can review in a later ticket).
- Do this for orga bcgov, bcdevops, and the private github org, filter by team, "Cloud-Pathfinder" or "Cloud-Pathfinder Technical" eg https://github.com/orgs/bcgov/teams followed by clicking on the Repositories tab https://github.com/orgs/bcgov/teams/cloud-pathfinder-technical/repositories 
- Set a quarterly calendar reminder to make new tickets to update the list every 3, 4 or 6 months (you decide)

.
.
.
.
.
.
.
.
.
.
.

**Extra Template Info**

***Definition of Ready***
These set of conditions will need to be met in order to bring a ticket into the sprint and start work. Protects the team from unclear requirements.
Issues (Task/Story/Spike) aligned to an EPIC and linked appropriately.
Assigned to the appropriate CPF MEMBER (and is not left blank in ZenHub) at the start of sprint (1st day of the sprint)
Acceptance has been defined for the issue and has been reviewed with CPF team and approved by the necessary approver.
Has been sized and estimated by the delivery team.
Detailed breakdown of the steps required to complete the story in the additional context
Any additional specifications have been documented, reviewed with CPF and approved by the necessary approver.

***Acceptance Criteria***
A set of pre-defined requirement that need to be met in order to mark the user story as “done”.
It should be testable with no room for interpretation
It should be either “pass” or “fail”
It should be clear enough for business stake holders to understand
As part of the user story, it should be written from the user perspective
A well written acceptance criteria is great for 
- managing expectations
- defining scope
- reducing ambiguity
- establishing testing criteria for QA
Given (Pre-condition) 
When (Action)
Then (Outcome)

***Definition of Done***
These set of conditions are met for work to be considered as complete on an issue.
All acceptance criteria(s) have been validated.
All the necessary documentation for the issue has been uploaded to Teams.
Functionality has been tested when applicable.
Functionality has been demonstrated to the relevant stakeholders (where applicable).
Story has been scheduled for a Sprint Demo/community update and impacted teams invited to the Demo.
Stories accepted by PO and documented for potential sharing with impacted users.
- Documented = Captured in Community Update (CU) deck
- Release notes = aspirational goal of providing a bulleted list of features and changes that our users can touch. Was going to store them in our Cloud Pathfinder repo. Can also display to users in the CU

**Comment by jon-mc-git** [comment]
Made some updates to the Team's Wiki but have some questions for Julian to move forward.  Julian is away currently so will put on hold as blocked and pick it up upon his return

**Comment by jon-mc-git** [comment]
Waiting on additional info from Julian on this one

**Comment by mitovskaol** [comment]
Since Wiki in Teams is going away, the content from Wiki will be migrated into SOF or technical docs.
